### PR TITLE
Badge should inherit color by default

### LINKED ISSUE #1007: Badge should inherit color by default

### PR DESCRIPTION
**Pull request by DenisVorop**
## PR includes

- [x] Bug Fix

## Related issues

Resolve #1007

### CODE REVIEW

**Review by tsumo — APPROVED** [review]

File: src/harmony/Badge/Badge.tsx [hunk 1]
```diff
@@ -24,24 +24,30 @@ type TextBadgeProps = Pick<
...
     iconRight?: React.ReactNode;
-    view?: 'default' | 'outline';
-    size?: 's' | 'm' | 'l' | 'xl';
+    view?: keyof typeof viewMap;
+    size?: keyof typeof sizeMap;
     text?: React.ReactNode;
     color?: string;
     as?: keyof AllowedHTMLElements;
     action?: 'dynamic' | 'static';
+    hoverable?: boolean;
```
Comment (asabotovich) [inline src/harmony/Badge/Badge.tsx:48]: interactive

**Review by asabotovich — APPROVED** [review]

**Pull request merged**